test/functional/importmulti.py

```diff
@@ -53,6 +53,19 @@ def run_test (self):
         watchonly_address = address['address']
         watchonly_timestamp = timestamp
 
+        self.log.info("Should not import an already imported address")
+        result = self.nodes[1].importmulti([{
+            "scriptPubKey": {
+                "address": address['address']
+            },
+            "timestamp": "now",
+        }])
+        assert_equal(result[0]['success'], True)
```

Comment:
The test below ("Should not import an address with private key if is already imported") seems good, but this test ("Should not import an already imported address") is confusing to me.

For one thing, it doesn't seem to have anything to do with the bug, because it passes both before and after the bugfix. But also the description seems misleading because it is checking that success is true while claiming the address shouldn't be imported.

Would suggest either dropping this new test, or moving it to a separate commit where it is clear this has nothing to do with the bugfix. And the test description should be clarified it it will be kept.

Reply:
I'm dropping this test.